recipes/tiled/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "tiled" %}
+{% set version = "0.1.0a60" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiled-{{ version }}.tar.gz
+  sha256: 32f3fee3583fe7e899d908d79319c00dd5c20bb5245c44519653c45d20cfc457
+
+build:
+  entry_points:
+    - tiled = tiled.commandline.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - aiofiles
+    - alembic
+    - anyio
+    - appdirs
+    - blosc
+    - cached-property; python_version < '3.8'
```

Comment:
```suggestion
    - cached-property  # [py<38]
```

Reply:
I don't see this suggestion was committed, that's why we are still having trouble with the syntax. Please accept the suggestion, or implement it on your side.